plugins/dynamix/include/DefaultPageLayout.php

```diff
@@ -235,6 +235,10 @@ function openWindow(cmd,title,height,width) {
   form.submit();
 }
 function openTerminal(tag,name,more,height,width) {
+  if (/MSIE|Edge/.test(navigator.userAgent)) {
```

Comment:
Isn't this going to trigger on both the old EdgeHTML based browser and the new chromium based one?